### PR TITLE
Ping Purger Integrated Into Image

### PR DESCRIPTION
**Pull request by drewjj**
This pull request integrates the internal CDOT ping purger into the publicly available rsu_ping_fetch image.
- Added to cron as a separate script to keep things isolated but simple
- Added unit tests for new feature code
- Added to README for quick explanation of the new feature and how to customize it

### CODE REVIEW

File: addons/images/rsu_ping_fetch/README.md [hunk 1]
```diff
@@ -2,15 +2,19 @@
...
-- [About ](#about-)
-- [Requirements ](#requirements-)
+- [RSU Ping Fetch](#rsu-ping-fetch)
+  - [Table of Contents](#table-of-contents)
+  - [About ](#about-)
+  - [Requirements ](#requirements-)
 
 ## About <a name = "about"></a>
 
 This directory contains a microservice that runs within the RSU Manager GKE Cluster. The rsu_ping_fetch application populates the CV Manager PostGreSQL database's 'ping' table with the current online statuses of all RSUs recorded in the 'rsus' table. These statuses are retrieved directly from a [Zabbix server](https://www.zabbix.com/).
```
Comment (dmccoystephenson) [inline addons/images/rsu_ping_fetch/README.md]: Should we change this to say 'CV Manager' instead of 'RSU Manager'?

**Review by dmccoystephenson — APPROVED** [review]
Looks good to me, just the one comment!

**Review by payneBrandon — APPROVED** [review]
Looks good to me!

File: addons/images/rsu_ping_fetch/README.md (hunk 1 above)
Comment (drewjj) [reply addons/images/rsu_ping_fetch/README.md]: Addressed

**Pull request merged**